app/src/main/java/com/onboarding/parkingsystemjava/mvp/view/ConfigureParkingView.java

```diff
@@ -0,0 +1,45 @@
+package com.onboarding.parkingsystemjava.mvp.view;
+
+import android.content.Context;
+import android.widget.Toast;
+import com.onboarding.parkingsystemjava.R;
+import com.onboarding.parkingsystemjava.databinding.ConfigureParkingLotsDialogFragmentMainActivityBinding;
+import com.onboarding.parkingsystemjava.fragment.ConfigureParkingLotsDialog;
+import com.onboarding.parkingsystemjava.mvp.contract.SetParkingContract;
+import java.lang.ref.WeakReference;
+
+public class ConfigureParkingView implements SetParkingContract.SetParkingView {
+    private final WeakReference<ConfigureParkingLotsDialog> fragmentRef;
```

Comment:
please put the weakReference into a FragmentView (just like ActivityView) in the base package

Reply:
we've discussed that with lucho and he told me it was unnecessary

Reply:
please, do it because if you need to use another fragment in the future you will have duplicated code

Reply:
done!